boards/common/stm32-stlink/Makefile.include

```diff
@@ -0,0 +1,14 @@
+# configure the serial terminal
+PORT_LINUX ?= /dev/ttyACM0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+# Default is to use ST-link v2-1 adapter (used by all Nucleos)
+export DEBUG_ADAPTER ?= stlink
+export STLINK_VERSION ?= 2-1
+
+# Include openocd adapters for st-link
+include $(RIOTMAKE)/tools/openocd-adapters/stlink.inc.mk
```

Comment:
there seems to be a include-loop: here we include `stlink.inc.mk` and below `openocd.inc.mk`. But `openocd.inc.mk` does also include `stlink.inc.mk`... So seems to me we can drop this line.